### ISSUE TITLE
classification stops after some time

### ISSUE THREAD

**Issue by janvda** [janvda/node-red-doorbell#2]
I think it is due to the sox command that stops after some time (around 1 minute)

```
sox  -q -t alsa default -L -e signed-integer -c 1 -r 44100 -b 16 -t raw - trim 0 60 gain 0
```

**Comment by janvda** [comment]
Fixed by setting the "Recording Duration" properties as follows:

![image](https://user-images.githubusercontent.com/17238398/97986470-08a46680-1dda-11eb-8653-d24bb2d2e80c.png)

**Comment by janvda** [comment]
fixed by 075da417d326e2d0a84af8c2a1e487d03ec7590b
